unison-core/src/Unison/ABT.hs

```diff
@@ -91,7 +91,6 @@ where
 import Control.Lens (Lens', lens, use, (%%~), (.=))
```

Comment:
The ormolu diffs here are unfortunate... looks like you are in the same boat as Arya:

- Your PATH ormolu is `0.4.0.0` (hence the indentation in the first place, from running the "ormolu everything" script
- Your HLS ormolu is older, so when you touch a file, this can happen.

Reply:
Yeah when I "ormolu'd everything" I was under the impression we'd want the most recent ormolu version so I explicitly used `0.4.0.0`, but that certainly turns out to not be the case. I suppose we could re-ormolu everything, but I'm also pretty content to just let it sort itself out over time. I've since deleted all my newer copies of ormolu haha.